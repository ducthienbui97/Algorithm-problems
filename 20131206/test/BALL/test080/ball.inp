9 434017 858168 459423 134353
365491 818944 406068 572139 910821 68387 468050 746950 944214 218035 240280 226077 211641 520769 863492 179383 449788 831796 138308 883194 228332 560881 800335 87112 922116 378228 342770 676782 848561 167103 384094 864005 447189 102373 966512 211718 127534 786292 314501 954983 883392 495032 9098 803133 242562 765171 494377 362264 525738 847522 948846 856762 968392 860366 646669 20305 170568 816857 563118 530401 641957 711634 496821 834319 548362 276538 340551 541665 888445 633262 868790 681702 645844 102196 144560 309531 977434 822652 904360 69422 979433 
950543 406623 503478 126416 919151 73040 353170 42624 810014 610324 317419 827755 798177 199315 702529 802677 831559 144500 491938 139317 250809 677502 753228 121355 342601 286295 591314 625157 999184 252559 54451 703274 811766 344236 632423 143891 74504 637633 832929 991493 563784 893801 525111 549712 237122 465919 205386 382783 649942 498487 516549 916453 756020 859158 351699 881106 664590 677320 318125 804843 194489 891201 411040 68968 925641 979712 175424 36970 294878 986672 633704 746738 35560 2107 748170 816580 859270 647532 891297 845229 46441 
613065 753449 441100 387122 849284 720973 750026 540426 585221 421419 795986 844091 843246 916738 974786 827842 371735 643795 13212 255557 604923 460786 284451 611295 671953 307337 932229 333117 901408 363433 151961 613151 641415 180118 735218 487588 818150 558754 434793 244673 423735 618968 58539 80396 78209 388910 151752 132880 463082 391705 881838 624090 959384 782795 525318 87085 382033 551637 28090 232604 860603 885852 623675 137815 644889 604651 215628 199016 870938 563460 703937 5343 794544 496377 935492 400917 74011 61188 906877 982943 19834 
277124 252232 728489 561800 155674 27980 853751 676106 205011 903488 714289 408484 19445 975615 511796 745964 8471 247200 14228 836445 174957 360535 174378 49848 47181 704687 282235 591053 651282 893316 490080 590749 362040 21432 432835 187193 612421 673813 75227 699611 184119 312588 280677 348611 643802 725854 749 461442 305462 830866 847380 692575 94825 696500 992056 77424 590790 747559 859302 98032 27297 261468 30969 389881 284525 157195 47545 132261 747115 78879 222363 787925 716479 670526 650604 30510 862732 126929 956420 336914 770329 960152 519507 669033 551028 167494 654858 185773 734699 610579 933682 308315 566339 74056 64378 512524 581968 198642 361952 371077 408630 783365 169281 225868 637801 250131 532063 872967 250138 993075 268627 492544 242465 698714 880154 60261 164091 736378 473181 894079 148439 212080 653539 461867 142206 981396 495497 334933 120765 965039 727857 729989 404971 988123 324377 384592 640943 13195 845214 981806 10315 15890 140422 261803 583941 979038 185501 37312 985682 931924 252019 181043 170139 851278 321852 875137 798854 346789 327386 834755 651541 304574 492120 116542 854375 458297 381738 910816 337033 28812 524929 463581 28865 779556 996150 614742 423291 154347 350124 630756 73654 752994 851377 892667 411422 915197 764635 235652 100275 179762 625444 429165 633800 545745 246866 934391 129524 654569 903319 687257 668165 860265 757008 138441 43881 985347 779279 312458 63972 561603 444517 75568 939785 489050 945792 249425 90897 672495 73456 705982 419079 927226 416222 470946 875235 961367 818996 259732 846738 425588 315653 60096 840123 316536 269520 556688 568615 818671 467267 705049 400605 349889 402134 66280 621524 879820 802754 156762 770878 43684 904650 698203 836750 135071 143328 567847 931681 303448 639366 690355 954740 755114 866212 145706 320505 286473 992929 728783 981560 27674 845756 701502 879572 432343 821331 788610 268968 824233 344711 740451 844813 253994 852882 871405 808483 587549 263025 172644 675853 194270 380638 864476 554492 624044 694329 19257 135471 661515 712502 78043 124088 755451 542271 170992 23087 539407 906825 268841 431868 855821 205168 511376 385248 927876 994671 558028 380524 46201 351365 95756 678184 665085 227714 630037 847405 491790 885952 248042 557191 615387 790402 415092 196160 876588 956690 146020 961930 482833 579107 839731 427935 475469 243814 411664 312200 19988 865801 222119 766234 14855 579972 272306 144015 316877 563894 64337 631237 318445 43641 227622 331154 975388 479537 777900 85284 450307 376111 640972 266064 813177 237833 463877 785403 978759 92968 277986 374845 233848 890285 165201 972212 403812 603034 256860 78249 887151 778451 258339 767657 40024 833040 138042 876755 332460 788145 425839 244776 963310 713218 837843 699431 974024 46954 857212 878574 408795 983833 430987 41145 631757 637973 932233 680301 394027 959015 348575 772205 309002 226576 858028 917389 315483 228258 528709 100216 675504 357973 667253 402353 691321 703886 462952 942168 822925 762130 80869 925658 487868 800154 337662 937253 944967 330598 913303 421051 421220 553353 451517 647211 431121 352052 997344 767242 747438 796645 315792 703973 155510 42318 188965 945076 665958 786907 590900 57967 320728 935257 822753 129443 230459 980281 997170 496417 682508 201083 544957 3120 317992 303547 531742 288230 37686 570890 509695 525107 44317 333892 43709 648679 398754 859272 427627 12781 710402 971806 609557 132011 133608 552065 672394 288674 345418 638872 466030 702411 415196 254796 78308 429297 513342 965368 152276 546722 282619 262502 706063 797575 189699 211076 232480 209444 391538 535684 772009 917772 514009 197280 186262 413760 303979 705298 306797 658714 964168 844446 581804 181287 667060 275166 185738 704818 576482 410306 894854 793479 239526 555244 82293 755313 544813 216600 237854 414693 386430 327102 820999 764184 309062 656834 243378 719584 932989 564929 947347 107085 845419 735574 364083 228882 595397 528693 819834 874407 443399 536066 62201 730413 201551 114936 227035 752125 664169 863721 100036 62864 643114 274858 790276 551955 263249 396648 987821 840816 220793 708196 431378 970232 299717 584646 889416 57428 45744 537753 203469 986821 444156 791214 786080 98283 747169 291247 65807 456746 918032 932133 490597 530817 374002 831746 152233 140698 365423 737798 73787 337821 646763 924113 280808 376564 728329 177553 828170 430141 438270 965249 509784 609385 476977 537655 832104 255212 982457 358812 385350 309258 220776 965680 971270 223120 103138 235933 144660 912650 574962 213665 418314 901130 862966 261661 107320 505628 130438 206270 598962 862380 756702 678994 355037 591182 334650 680250 212039 842274 746079 554561 94756 769700 332215 616886 313169 186602 252093 12126 734696 66185 323024 918806 645055 763843 103369 557584 614943 398680 29606 181541 799396 751354 841925 563767 547142 670058 713206 402556 159757 376197 702150 449779 540461 511217 131960 119973 227611 140288 484501 675659 887529 486306 361355 281695 483611 590868 586818 243305 946003 277204 568910 149801 352649 53779 