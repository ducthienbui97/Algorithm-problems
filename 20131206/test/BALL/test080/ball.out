14070876081073
